docs/src/pages/landing-page/Index.vue

```diff
@@ -0,0 +1,148 @@
+<template>
+  <q-page class="bg-lp-dark text-white relative-position">
+
+    <section class="q-px-lg">
+      <div class="row justify-end socials q-pa-md">
+        <q-btn flat :icon="fabGithub" color="lp-primary" class="socials-item" />
+        <q-btn flat icon="message" color="lp-primary" class="socials-item" />
+        <q-btn flat icon="forum" color="lp-primary" class="socials-item" />
+        <q-btn flat :icon="fabTwitter" color="lp-primary" class="socials-item" />
+        <q-btn
+          flat
+          :icon="fabFacebookSquare"
+          color="lp-primary"
+          class="socials-item"
+        />
+      </div>
+    </section>
+
+    <section>
+      <div class="row flex flex-center q-mt-xl">
+        <div class="col-6 column items-center text-center about">
+          <div class="about-img">
+            <div class="bg-red" style="height: 250px; width: 250px;">
+              QUASAR logo 😀
+            </div>
+          </div>
+
+          <div class="q-py-lg text-lp-light about__text">
+            The open source multi-platform development framework based on Vue.js
+            with an enterprise vocation.
+          </div>
+
+          <div>
+            <q-btn
+              color="lp-accent"
+              label="Take a look across the stars"
+              class="shadow-6 lp-btn lp-btn-primary"
+            />
+          </div>
+
+          <div class="q-mt-lg">
+            <q-btn
+              flat
+              round
+              icon="arrow_downward"
+              class="text-white"
+              size="lg"
+              padding="none"
+            />
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <section>
+      <div class="sponsors row justify-center q-mt-xl q-pt-sm">
+        <div class="col-7 column items-center">
+          <div class="text-lp-primary text-center sponsors-heading">
+            Our platform sponsors
+          </div>
+
+          <div class="row bg-primary q-my-lg">
+            Sponsor Logos 😄
+          </div>
+
+          <div class="q-mb-xl">
+            <q-btn
+              flat
+              padding="none"
+              label="Full sponsors' list"
+              text-color="lp-light"
+              no-caps
+              class="lp-btn-underline"
+            />
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <section></section>
+  </q-page>
+</template>
+
+<script lang="ts">
+import { defineComponent } from 'vue'
+
+import {
+  fabGithub,
+  fabTwitter,
+  fabFacebookSquare
+} from '@quasar/extras/fontawesome-v5'
+
+export default defineComponent({
+  name: 'Index',
+  setup () {
+    return {
+      fabGithub,
+      fabTwitter,
+      fabFacebookSquare
+    }
+  }
+})
+</script>
+
+<style scoped lang="scss">
+.socials {
+  &-item {
+    padding: 0px;
+    font-size: 14px;
+    margin-left: 10px;
+  }
+}
+
+.about {
+  &__text {
+    font-size: 20px;
+    color: rgba(255, 255, 255, 0.54);
```

Comment:
Use this SCSS addition (to be called into `css/landing-page/variables`) to have easier to use utility classes

```scss
$white: #fff;
$black: #000;

$additonal-colors: (
  'lp-primary' $lp-primary,
  'lp-secondary' $lp-secondary,
  'lp-accent' $lp-accent,
  'lp-positive' $lp-positive,
  'lp-negative' $lp-negative,
  'lp-info' $lp-info,
  'lp-warning' $lp-warning,
);

@each $color-name, $color in $svg-fill-colors {
  .bg-#{$color-name} {
    background-color: $color !important;
  }

  .text-#{$color-name} {
    color: $color !important;
  }
}

$alpha-map: (
  '87' 0.87,
  '54' 0.54,
  '38' 0.38,
  '28' 0.28,
  'focused-dark' 0.24,
  'focused-light' 0.12,
  '12' 0.12,
  'activated-dark' 0.24,
  'activated-light' 0.12,
  'selected' 0.08,
  'hover' 0.04
);

$color-map: (
  'lp-primary' $lp-primary,
  'lp-secondary' $lp-secondary,
  'lp-accent' $lp-accent,
  'black' $black,
  'white' $white,
  'lp-positive' $lp-positive,
  'lp-negative' $lp-negative,
  'lp-info' $lp-info,
  'lp-warning' $lp-warning,
);

@each $alpha-name, $alpha in $alpha-map {
  @each $color-name, $color in $color-map {
    .bg-#{$color-name}-#{$alpha-name}.bg-#{$color-name}-#{$alpha-name} {
      background-color: rgba($color, $alpha);
    }

    .text-#{$color-name}-#{$alpha-name}.text-#{$color-name}-#{$alpha-name} {
      color: rgba($color, $alpha);
    }
  }
}
```

I also started adding utility classed for font-size lately like

```scss
.text-size {
  &-12 {
    font-size: 12px;
  }

  &-16 {
    font-size: 16px;
  }

  &-18 {
    font-size: 18px;
  }
}
```

You can easily generate them dynamically as I did for the colors